diagnostic.md

```diff
@@ -94,4 +94,5 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'hobby' = exercise
+'name' = ('Jason', 'Weeks')
```

Comment:
This question has 3 parts. Please read the question carefully. The value of teamMember['name'] is actually the entire object, which would look like this: 
teamMember['name']['surname'] = {given: 'Jason', surname: 'Weeks'}
The 3rd part of the question, teamMember['name']['surname'], would return 'Weeks'.